src/components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -86,59 +86,62 @@ const AudioPlayer = () => {
   }, [audioPlayerElRef, onAudioPlay, onAudioPause, onAudioEnded, onAudioLoaded]);
 
   return (
-    <div
-      className={classNames(styles.container, styles.containerDefault, {
-        [styles.containerHidden]: isHidden,
-        [styles.containerMinimized]: isMobileMinimizedForScrolling,
-      })}
-    >
-      <div className={styles.innerContainer}>
-        {/* We have to create an inline audio player and hide it due to limitations of how safari requires a play action to trigger: https://stackoverflow.com/questions/31776548/why-cant-javascript-play-audio-files-on-iphone-safari */}
-        <audio
-          src={audioData?.audioUrl}
-          style={{ display: 'none' }}
-          id="audio-player"
-          ref={audioPlayerElRef}
-        />
-        <AudioKeyBoardListeners
-          seek={(seekDuration) => {
-            triggerSeek(seekDuration);
-          }}
-          togglePlaying={() => togglePlaying()}
-          isAudioPlayerHidden={isHidden}
-        />
-        {reciterId && audioData?.chapterId && (
-          <QuranReaderHighlightDispatcher
-            audioPlayerElRef={audioPlayerElRef}
-            reciterId={reciterId}
-            chapterId={audioData?.chapterId}
+    <>
+      <div className={styles.emptySpacePlaceholder} />
+      <div
+        className={classNames(styles.container, styles.containerDefault, {
+          [styles.containerHidden]: isHidden,
+          [styles.containerMinimized]: isMobileMinimizedForScrolling,
+        })}
+      >
+        <div className={styles.innerContainer}>
+          {/* We have to create an inline audio player and hide it due to limitations of how safari requires a play action to trigger: https://stackoverflow.com/questions/31776548/why-cant-javascript-play-audio-files-on-iphone-safari */}
+          <audio
+            src={audioData?.audioUrl}
+            style={{ display: 'none' }}
+            id="audio-player"
+            ref={audioPlayerElRef}
           />
-        )}
-        {reciterId && audioData?.chapterId && (
-          <AudioRepeatManager
-            audioPlayerElRef={audioPlayerElRef}
-            reciterId={reciterId}
-            chapterId={audioData?.chapterId}
+          <AudioKeyBoardListeners
+            seek={(seekDuration) => {
+              triggerSeek(seekDuration);
+            }}
+            togglePlaying={() => togglePlaying()}
+            isAudioPlayerHidden={isHidden}
```

Comment:
Is this intentional?

Reply:
![image](https://user-images.githubusercontent.com/12745166/136681765-b7faa341-1c4f-41f6-bdb0-5e2416da28ce.png)

seems to be something to do with Github diff things. looks fine on vscode